src/components/DurationDropdown/DurationDropdown.tsx

```diff
@@ -29,16 +32,26 @@ type DurationDropdownProps = BasicDurationDropdownProps & {
 };
 
 export class DurationDropdown extends React.Component<DurationDropdownProps> {
+  static defaultPrefix = 'Last';
+  private prefixedDurations: Durations;
+
+  constructor(props) {
+    super(props);
+    const prefix = !!this.props.prefix ? this.props.prefix : DurationDropdown.defaultPrefix;
```

Comment:
shouldn't `this.props.prefix || DurationDropdown.defaultPrefix` work?

Reply:
I think this one is just a personal preference.  I've gotten into the habit of using !! and !!! to explicitly show that a non-boolean variable is truthy or falsey, respectively.  Let me know if there is a technical reason to avoid this approach.